lib/board.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+# Board class
+class Board
+  attr_accessor :board
+
+  def initialize
+    @board = [1, 2, 3, 4, 5, 6, 7, 8, 9]
+  end
+
+  def draw_board
+    puts "\n"
+    puts " #{@board[0]} | #{@board[1]} | #{@board[2]}"
+    puts '-----------'
+    puts " #{@board[3]} | #{@board[4]} | #{@board[5]}"
+    puts '-----------'
+    puts " #{@board[6]} | #{@board[7]} | #{@board[8]}"
+  end
+
+  def win_hor
```

Comment:
This logic works but is too hardcoded. In programming, we try not to repeat ourselves as much as possible. You checkout methods like `any, all, uniq, size [1..3], etc ` to make your code dryer.  This is for the **hor, dg and ver**
methods